project/Dependencies.scala

```diff
@@ -15,7 +15,10 @@ object Dependencies {
     "uk.gov.hmrc" %% "play-reactivemongo" % "6.4.0",
     "com.github.pureconfig" %% "pureconfig" % "0.10.2",
     "uk.gov.hmrc" %% "http-caching-client" % "8.0.0",
-    "uk.gov.hmrc" %% "microservice-bootstrap" % "10.6.0",
+    "uk.gov.hmrc" %% "bootstrap-play-25" % "1.1.0",
+    "uk.gov.hmrc" %% "play-authorisation" % "5.2.0",
```

Comment:
`play-authorisation` seems not to be needed. `authParamsControllerConfig` which depends on it is not being used.